core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/VideoSizeChangedForwarder.java

```diff
@@ -0,0 +1,52 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import android.view.Surface;
+
+import com.google.android.exoplayer2.Format;
+import com.google.android.exoplayer2.decoder.DecoderCounters;
+import com.google.android.exoplayer2.video.VideoRendererEventListener;
+import com.novoda.noplayer.listeners.VideoSizeChangedListeners;
+
+class VideoSizeChangedForwarder implements VideoRendererEventListener {
+
+    private final VideoSizeChangedListeners videoSizeChangedListeners;
+
+    VideoSizeChangedForwarder(VideoSizeChangedListeners videoSizeChangedListeners) {
+        this.videoSizeChangedListeners = videoSizeChangedListeners;
+    }
+
+    @Override
+    public void onVideoSizeChanged(int width, int height, int unappliedRotationDegrees, float pixelWidthHeightRatio) {
+        videoSizeChangedListeners.onVideoSizeChanged(width, height, unappliedRotationDegrees, pixelWidthHeightRatio);
+    }
+
+    @Override
+    public void onVideoEnabled(DecoderCounters counters) {
+        //TODO should we send ?
```

Comment:
OOOOoooo more! We need to figure out the mapping between `ExoPlayer` and `no-player`.